elasticsearch/files/elasticsearch.yml

```diff
@@ -430,3 +430,9 @@ discovery.zen.ping.unicast.hosts: [{% for member in server.cluster.members %}"{{
 #monitor.jvm.gc.old.warn: 10s
 #monitor.jvm.gc.old.info: 5s
 #monitor.jvm.gc.old.debug: 2s
+
+{%- if server.get("enable_script", False) %}
+script:
+  indexed: on
+  inline: on
```

Comment:
Can you try to be less permissive and enable only the required extensions? See https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-scripting-security.html.